storage/innobase/xtrabackup/doc/source/xtrabackup_bin/xbk_option_reference.rst

```diff
@@ -154,6 +154,61 @@ Options
    :option:`xtrabackup --defaults-group` option. It is needed for
    ``mysqld_multi`` deployments.
 
+.. option::  --dump-innodb-buffer-pool
+
+   This option controls whether or not a new dump of buffer pool
+   content should be done.
+
+   If this option is set to `ON`, the file :file:`ib_buffer_pool` is copied as
+   soon as the backup starts provided the status variable
+   ``innodb_buffer_pool_dump_status`` reports that the dump has been
+   completed.
+
+   .. code-block:: mysql
+
+      SET GLOBAL  innodb_buffer_pool_dump_now=ON
```

Comment:
This code showing how start buffer dump manually. Instead, we provided xtrabackup option:
```
xtrabackup --backup --dump-innodb-buffer-pool --target-dir=/home/user/backup
```